centreon/src/Core/User/Application/UseCase/FindUserPermissions/FindUserPermissionsResponse.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * Copyright 2005 - 2024 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types = 1);
+
+namespace Core\User\Application\UseCase\FindUserPermissions;
+
+use Core\Application\Common\UseCase\StandardResponseInterface;
+use Core\User\Domain\Model\Permission;
+
+final readonly class FindUserPermissionsResponse implements StandardResponseInterface
+{
+    /**
+     * @param Permission[] $permissions
+     */
+    public function __construct(public readonly array $permissions)
+    {
+    }
+
+    public function getData(): mixed
```

Comment:
self no ? because it's return $this

```suggestion
    public function getData(): self
```

Reply:
ok